java/test/jmri/jmrix/roco/RocoXNetThrottleTest.java

```diff
@@ -60,91 +60,91 @@ public void testCtorWithArg() throws Exception {
 
     // Test the initilization sequence.
     @Override
-    @Ignore("parent class method creates a new throttle")
```

Comment:
lol of these ignores you are removing were intentional because empty tests add to the count of number of tests run and the number of tests that succeed.  Ignored tests don’t add to either of those counts.  

I am actually interested in those statistics, so removing the ignores throws off the numbers I am looking at.